### ISSUE TITLE
Multiline annotation

### ISSUE THREAD

**Issue by racinmat** [slevomat/coding-standard#199]
When having multiline annotation

```
	/**
	 * @Route("/configs/{config}/domains/{domain}/locales/{locale}/messages", name="jms_translation_update_message",
	 *     defaults = {"id" = null}, options = {"i18n" = false}, methods={"PUT"})
	 */
	public function updateMessageAction(Request $request, string $config, string $domain, string $locale): Response {
	}
```
TypeHintDeclarationSniff says:

  `Method \AppBundle\Controller\TranslateController::updateMessageAction() does not need documentation comment.
         (SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff)`

But when annotation is single line, 
```
	/**
	 * @Route("/configs/{config}/domains/{domain}/locales/{locale}/messages", name="jms_translation_update_message", defaults = {"id" = null}, options = {"i18n" = false}, methods={"PUT"})
	 */
	public function updateMessageAction(Request $request, string $config, string $domain, string $locale): Response {
	}
```
TypeHintDeclarationSniff does not complain. 
I have $usefulAnnotations set to `["@todo", "@dataProvider", "@Route", "@ORM\PreUpdate", "@ORM\PostLoad", "@Assert\Callback"]` and I did not find any option to allow multiline annotations.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/b0ef7275776e8588dd88152838ae944fce4d2c97
